Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -942,7 +942,7 @@
     <!-- XHED: Page headline for the positive result additional warning page-->
     <string name="submission_positive_other_warning_headline">"Please help all of us!"</string>
     <!-- YTXT: Body text for the positive result additional warning page-->
-    <string name="submission_positive_other_warning_body">"Next, you can make sure that the Corona-Warn-App shares your random IDs of the last 14 days with others. By doing this, you can warn other people and help to break the infection chain.\n\nSince only impersonal random IDs are transmitted, your identity will remain anonymous."</string>
```

Comment:
Please don't delete it, just leave it like it is if a text is updated. It will be overwritten with the next translation delivery, if you change something here it could cause merge conflicts.

Reply:
Ok, will add it back